components/SearchBar.tsx

```diff
@@ -1,36 +1,50 @@
-import { StyleSheet, TextInput, View } from "react-native";
+import { StyleSheet, TextInput, View, TouchableOpacity } from "react-native";
 import React, { useState } from "react";
 import StatusBarSpace from "./StatusBarSpace";
 import { useNavigation } from "@react-navigation/core";
+import { Feather } from "@expo/vector-icons";
+import colors from "../data/colors";
 
 export default function SearchBar() {
   const navigation = useNavigation();
   const [text, setText] = useState("");
-
   return (
     <View style={{ backgroundColor: "#dcb688" }}>
       <StatusBarSpace />
-      <TextInput
-        value={text}
-        onChangeText={setText}
-        placeholder="Search"
-        style={{
-          margin: 7,
-          paddingHorizontal: 10,
-          paddingVertical: 5,
-          borderRadius: 10,
-          backgroundColor: "white",
-          elevation: 10,
-        }}
-        onSubmitEditing={() => {
-          navigation.navigate(
-            "Home" as never,
-            { screen: "ProductsScreen", params: { search: text } } as never
-          );
-          setText("");
-        }}
-      />
+      <View style={{ flexDirection: "row", justifyContent: "center" }}>
```

Comment:
Wanted to get some clarification on this part. Will this appear all the time next to the search bar or only within the products and product detail screens?

Reply:
It is there in all screens except the cart and check out page

Reply:
I corrected your suggestions.

Reply:
Awesome! Great work.